src/components/graphs/bargraph/BarGraph.js

```diff
@@ -20,23 +62,50 @@ const def = {
 
 const BarGraph = ({ count }) => {
   const [barData, setBarData] = useState(def);
-  const [incidentCount, setIncidentCount] = useState(null);
+  const [horizBarData, setHorizBarData] = useState(def);
+  const [horizShowing, setHorizShowing] = useState(false);
 
   useEffect(() => {
-    setIncidentCount(count);
-  }, [count]);
+    const queryMatcher = window.matchMedia('(max-width: 1100px)');
+    setHorizShowing(queryMatcher.matches);
+    const callback = ({ matches }) => {
+      setHorizShowing(matches);
+    };
+    queryMatcher.addListener(callback);
+
+    return () => {
+      queryMatcher.removeListener(callback);
+    };
+  }, []);
```

Comment:
Considering this is the only place this is being used, it was reasonable to move it directly into the file it'd be used in - nice find!